src/ts/lib/storage.ts

```diff
@@ -108,6 +108,48 @@ class Storage {
 		this.setSpace(obj);
 	};
 
+	setLastOpened (windowId: string, param: any) {
+		const obj = this.get('lastOpened') || {};
+		obj[windowId] = Object.assign(obj[windowId] || {}, param);
+		this.set('lastOpened', obj, true);
+	};
+
+	deleteLastOpenedByObjectId (objectIds: string[]) {
+		const obj = this.get('lastOpened') || {};
+
+		let windowIdsToDelete = Object.keys(obj).reduce(
+			(windowIds, windowId) => !obj[windowId] || objectIds.includes(obj[windowId].id) ? windowIds.concat(windowId) : windowIds, []
+		);
+
+		this.deleteLastOpenedByWindowId(windowIdsToDelete, true);
+	};
+
+	deleteLastOpenedByWindowId (windowIdsToDelete: string[], homeIncluded?: boolean) {
+		if (windowIdsToDelete.length == 0) {
+			return;
+		};
+
+		const obj = this.get('lastOpened') || {};
+
+		if (!homeIncluded) {
+			windowIdsToDelete = windowIdsToDelete.filter((id) => id !== '1');
```

Comment:
wrong formatting 
```
id => id != 1
```

Reply:
Are you suggesting to change the stored type to number instead of string?